### PR TITLE
Fix the issue of unable to click on QNX platform.

### PR DESCRIPTION
**Pull request by wangzhenGitHup**
Fix the issue of unable to click on QNX platform.

### CODE REVIEW

File: pal/system-info/native/system-info.ts [hunk 1]
```diff
@@ -88,7 +89,7 @@ class SystemInfo extends EventTarget {
 
         this.platform = platformMap[__getPlatform()];
         // eslint-disable-next-line max-len
-        this.isMobile = this.platform === Platform.ANDROID || this.platform === Platform.IOS || this.platform === Platform.OHOS || this.platform === Platform.OPENHARMONY;
+        this.isMobile = this.platform === Platform.ANDROID || this.platform === Platform.IOS || this.platform === Platform.OHOS || this.platform === Platform.OPENHARMONY || this.platform === Platform.QNX;
```
Comment (minggo) [inline pal/system-info/native/system-info.ts:92]: QNX is not a mobile platform.

File: pal/system-info/native/system-info.ts [hunk 2]
```diff
@@ -56,7 +56,8 @@ const platformMap: Record<number, Platform> = {
     // 5 is IPAD
     5: Platform.IOS,
     6: Platform.OHOS,
-    7: Platform.OPENHARMONY,
+	7: Platform.QNX,
+    10: Platform.OPENHARMONY,
```
Comment (qiuguohua) [inline pal/system-info/native/system-info.ts]: image
The value of qnx is 8?

File: pal/system-info/native/system-info.ts (hunk 1 above)
Comment (qiuguohua) [reply pal/system-info/native/system-info.ts:92]: 'Platform.QNX' is a string, defined in pal/system-info/enum-type/platform.ts.

Comment (wangzhenGitHup) [reply pal/system-info/native/system-info.ts:92]: The QNX device being adapted has some mobile platform properties, such as touchscreen capabilities.

Comment (minggo) [reply pal/system-info/native/system-info.ts:92]: Touch is not a criterion for judging mobile devices. Does it not work properly without being set as a mobile device?

**Comment by wangzhenGitHup** [comment]
Currently, QNX is adapted to version3.5.0 of the engine. If there is a need to adapt to a higher version of the engine in the future, we will consider it then.

**Comment by minggo** [comment]
Why close it?

**Comment by wangzhenGitHup** [comment]
> Why close it?

Currently, the QNX adaptation engine version is 3.5.0. According to Guohua, this test is using version 3.8.Therefore, we will consider adapting to higher versions of QNX in the future.

**Pull request closed without merging**